clang/lib/Format/TokenAnnotator.cpp

```diff
@@ -1358,6 +1358,8 @@ class AnnotatingParser {
           Line.First->startsSequence(tok::kw_export, Keywords.kw_module) ||
           Line.First->startsSequence(tok::kw_export, Keywords.kw_import)) {
         Tok->setType(TT_ModulePartitionColon);
+      } else if (Line.First->is(tok::kw_asm)) {
+        Tok->setType(TT_InlineASMColon);
```

Comment:
Does the `else if` block (lines 1430-1436) at the end of this `case tok::colon` become redundant now? It should be removed if yes.

Reply:
Yes it appears to be redundant, I have refactored it out.